selfdrive/car/toyota/carstate.py

```diff
@@ -144,6 +144,11 @@ def update(self, cp, cp_cam):
       ret.leftBlindspot = (cp.vl["BSM"]["L_ADJACENT"] == 1) or (cp.vl["BSM"]["L_APPROACHING"] == 1)
       ret.rightBlindspot = (cp.vl["BSM"]["R_ADJACENT"] == 1) or (cp.vl["BSM"]["R_APPROACHING"] == 1)
```

Comment:
If you are reading the same signals I think you simplify it like this
```suggestion
      ret.rightBlindspot = (cp.vl["BSM"]["R_ADJACENT"] == 1) or (cp.vl["BSM"]["R_APPROACHING"] == 1)
      # TSS-P C-HR has flipped BSM signal
      if self.CP.carFingerprint in (CAR.CHR, CAR.CHRH):
        ret.leftBlindspot, ret.rightBlindspot = ret.rightBlindspot, ret.leftBlindspot
```